.github/workflows/continuous-integration.yaml

```diff
@@ -43,7 +43,9 @@ jobs:
         # make sure it exists before chown
         mkdir -p /tmp/m2
         chown -R nobody:nobody . /tmp/m2
-        su --shell /bin/bash --preserve-environment --command="source /usr/bin/mvn_test.sh && mvn_test" nobody
+        # we have to run as a non-root user to pass the spma tests
+        # secondly, we first download all maven dependencies and then run the tests because it fails with hanging downloads otherwise.
+        su --shell /bin/bash --preserve-environment --command="source /usr/bin/mvn_test.sh && mvn_run \"dependency:resolve $MVN_ARGS\" && mvn_test" nobody
```

Comment:
Is this the best way to do this? From the `su(1)` man page it suggests runuser is better?

       su is mostly designed for unprivileged users, the recommended solution
       for privileged users (e.g. scripts executed by root) is to use non-set-
       user-ID  command  runuser(1)  that  does not require authentication and
       provide separate PAM configuration.

Reply:
I've switched to `runuser`